domain-models-runtime/src/main/java/org/folio/rest/persist/PostgresClient.java

```diff
@@ -142,6 +143,8 @@ public class PostgresClient {
   /** map (Vertx, String tenantId) to PostgresClient */
   private static final MultiKeyMap<Object, PostgresClient> CONNECTION_POOL =
       MultiKeyMap.multiKeyMap(new HashedMap<>());
+  private static final MultiKeyMap<Object, PostgresClient> CONNECTION_POOL_READER =
+      MultiKeyMap.multiKeyMap(new HashedMap<>());
```

Comment:
CONNECTION_POOL_READER is not needed because `PostgresClient` is never read-only, it always support both read-only and read+write.
Instead a PG_POOLS_READER is needed because we need read-only `PgPool`s.